src/tools/rbd_ggate/CMakeLists.txt

```diff
@@ -5,5 +5,5 @@ add_executable(rbd-ggate
   debug.cc
   ggate_drv.c
   main.cc)
-target_link_libraries(rbd-ggate geom librbd librados global)
+target_link_libraries(rbd-ggate geom librados-cxx global)
```

Comment:
just want to double check with you -- does this compile? as i found that `main.cc` is using librbd APIs. without linking against librbd, `ld` will still fails, i guess.